Pearson-Advance/edx-platform
# PADV 916 - CCX support for Gradebook

## Ticket

https://agile-jira.pearson.com/browse/PADV-916

## Description

Due to necessity, we found that the gradebook was not working for CCXs after migration. This PR aims to add CCX compatibility to the gradebook.

## Changes

- Modify gradebook API authorization to allow CCXs.

## How to test?

- Setup devstack
- Checkout to this branch
- Create a CCX (licensed or not)
- Verify the access to the gradebook with a CCX coach, Instructor or Staff user.

## Screenshots

![Screenshot from 2023-12-26 17-20-04](https://github.com/Pearson-Advance/edx-platform/assets/62396424/f1b6ae3b-0a8e-4c87-82ff-b45ac63808dc)
